patches/chrome-BUILD.gn.patch

```diff
@@ -57,7 +57,7 @@ index ec09d0836a976ff1c4c005a928e7feb51640a381..1a864d9c1b044c08aae67001aeb4eb52
  
    compile_entitlements("entitlements") {
      entitlements_templates = [ "app/app-entitlements.plist" ]
-+    if (is_official_build) { entitlements_templates += [ "app/app-entitlements-chrome.plist" ] }
++    if (is_official_build) { entitlements_templates += [ "app/app-entitlements-chrome.plist", "//brave/app/app-entitlements-brave.plist" ] }
```

Comment:
this should be `entitlements_templates += brave_entitlements_templates`

Reply:
@bridiver pushed https://github.com/brave/brave-core/pull/9818